test/fixtures/test-runner/NODE_TEST_WORKER_ID/1.test.js

```diff
@@ -0,0 +1,7 @@
+// This file tests the `NODE_TEST_WORKER_ID` feature.
```

Comment:
Please rename the directory from `NODE_TEST_WORKER_ID` to `worker-id`.

Reply:
@cu8code this still needs to be addressed 😁